src/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import Sum from './components/Sum/Sum';
 import './App.scss';
 
 const App = () => (
-  <h1>React sum</h1>
+  <main className="main">
+    <h1>React sum</h1>
+    <Sum
+      a={1}
+      b={2}
+    />
```

Comment:
```suggestion
    <Sum a={1} b={2} />
```
no need to put every prop on the new line, if they don'y consume too much space